src/transformers/training_args.py

```diff
@@ -630,6 +634,9 @@ class TrainingArguments:
         },
     )
     no_cuda: bool = field(default=False, metadata={"help": "Do not use CUDA even when it is available"})
+    use_mps_device: bool = field(
+        default=False, metadata={"help": "Whether to use Apple Silicon M1 chip based `mps` device."}
```

Comment:
Removed "M1" since this is more general. For instance, it still works on their newer M2 machines.

```suggestion
        default=False, metadata={"help": "Whether to use Apple Silicon chip based `mps` device."}
```